test/suite.bats

```diff
@@ -3,14 +3,15 @@
 load helper
 
 setup() {
+  current_context=$(kubectl config current-context)
```

Comment:
Command substitution should be enclose by double quotes.

```bash
current_context="$(kubectl config current-context)"
```

Reply:
Done